### PR TITLE
Feat/sc 3377/sportsdataio revamp

### PR DESCRIPTION
**Pull request by vnavascues**

### CODE REVIEW

File: packages/sources/sportsdataio-linkpool/schemas/env.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/sportsdataio-linkpool-adapter.json",
+  "title": "Chainlink External Adapter for Sportsdataio (LinkPool version)",
+  "description": "Adapter got get data from Sportsdata.io",
+  "required": [],
```
Comment (1marcghannam) [inline packages/sources/sportsdataio-linkpool/schemas/env.json:5]: Missing key here?

File: packages/sources/sportsdataio-linkpool/src/endpoint/schedule.ts [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            cause: error,
+            url: join(options.baseURL, response.request.path),
+          })
+        }
+        return gameCreate
+      })
+    const encodedGameCreateList = gameCreateList.map((gameCreate) => {
+      let encodedGameCreate: string
+      try {
+        encodedGameCreate = utils.solidityPack(marketResultEncode.get(market) as string[], [
```
Comment (1marcghannam) [inline packages/sources/sportsdataio-linkpool/src/endpoint/schedule.ts:128]: 👏 🔥

File: packages/sources/sportsdataio-linkpool/test-payload.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+ "requests": [{}]
```
Comment (1marcghannam) [inline packages/sources/sportsdataio-linkpool/test-payload.json:2]: Remove or keep file?

**Review by 1marcghannam — APPROVED** [review]
Thank you Victor! I just left comments. 🙏

File: packages/sources/sportsdataio-linkpool/test-payload.json (hunk 3 above)
Comment (vnavascues) [reply packages/sources/sportsdataio-linkpool/test-payload.json:2]: keep it

File: packages/sources/sportsdataio-linkpool/schemas/env.json (hunk 1 above)
Comment (vnavascues) [reply packages/sources/sportsdataio-linkpool/schemas/env.json:5]: yes now, but once it supports multiple sports it won't (only the specific sport one)

**Pull request closed without merging**